Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{HOMEBREW_CACHE}/bazel-workdir"
```

Comment:
Ok that's because of the bazel run underneath for generating the bazel-complete.bash. We need to fix it upstream.